app/views/docs/offline.phtml

```diff
@@ -4,17 +4,57 @@
+    Offline Support can be enabled by setting <code>setOffilnePersistency</code> to true on your SDK's client.
+</p>
+
+<div class="notice">
+  <h2>Adding Offline Support to Existing Projects</h2>
+  <p>Enabling Offline Support introduces breaking changes to the behavior of Client SDKs. Asynchronous operations may not resolve until the device is online again. Existing projects will have to be updated.</p>
+  <p><a href="#async-operation">Learn more about asynchronous operations</a></p>
+</div>
+
+[TODO: Code Example]
+<p>
+    Offline Support will cache all read request in a <a href="https://en.wikipedia.org/wiki/Cache_replacement_policies#LRU" target="_blank" rel="noopener">Least Recently Used (LRU)</a> cache, so the information can remain available even with no internet connection. The LRU cache defaults to a size of 16MB and can be configured using the <code>setOfflineCacheSize</code> method.
 </p>
 
-<h3>Considerations and Tradeoffs</h3>
+[TODO: Code Example]
+
+<h2 href="#async-operation" id="async-operation">Asynchronous Operation</h2>
 <p>
-    Offline Support adds a cache layer that takes up additional resources and adds overhead to your requests. This may not be desirable for some applications, especially if it's not expected to be used without a stable internet connection.
+    When Offline Support is enabled, asynchronous write operations only resolve after the server accepts the request. Requests made with the Client SDK may block code execution until network connection is available again.
 </p>
-<h3>Offline Support for Existing Projects</h3>
+[TODO: Show UML diagram]
+
+<p>
+    For example, the code example below will <b>block code execution</b> when the device is offline.
+</p>
+
+[TODO: Show Example, in comments clearly mark that it's ]
+
 <p>
-    Asynchronous operations may not resolve immediately when Offline Support is enabled. Existing projects will require some code to be rewritten when adopting Offline Support. <a href="TBD">Learn more about how to handle asynchronous operation</a>.  
-</p>
\ No newline at end of file
+    To avoid blocking code execution, asynchronous operations should be handled [TODO: fill description]
+</p>
+
+[TODO: Show correct version of example. ]
+
+<h2>Conflict Resolution</h2>
+<p>
+    With Offline Support enabled, your app can continue sending requests to Appwrite APIs while offline through a cache when offline. Read requests will be fetched from the local cached data, and write requests will be queued and executed once the device is online again. This can lead to write conflicts when the device is online again.
+</p>
+<p>
+    [TODO: Describe resolution behavior + tips if available.]
```

Comment:
When a locally queued request is sent to the server, it will include the timestamp when the request was queued in the `X-Appwrite-Timestamp` header. Appwrite will compare that request timestamp with the document's `$updatedAt` timestamp and return a status 409 error if the document's `$updatedAt` timestamp is newer than the request timestamp. The SDK will revert the changes in the local cache, and then the client can try the request again.